### PR TITLE
Sitemap improvements

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: static/robots.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 User-agent: *
 Disallow: /experimental/
+sitemap: https://pub.dartlang.org/sitemap.txt
```
Comment (kevmoo) [inline static/robots.txt]: See https://developers.google.com/search/reference/robots_txt#sitemap

File: app/lib/frontend/backend.dart [hunk 2]
```diff
@@ -77,9 +77,12 @@ class Backend {
   }
 
   /// Retrieves the names of all packages, ordered by name.
-  Stream<String> allPackageNames() {
+  Stream<String> allPackageNames({DateTime updatedSince}) {
     final query = db.query(models.Package);
-    return query.run().map((p) => (p as models.Package).name);
+    return query.run().where((p) {
+      return updatedSince == null ||
```
Comment (isoos) [inline app/lib/frontend/backend.dart]: why not put this into the query?

Comment (kevmoo) [reply app/lib/frontend/backend.dart]: Oh!...uh...I guess I could. Wasn't sure how...

Comment (kevmoo) [reply app/lib/frontend/backend.dart]: Done!

**Pull request merged**